animation_nodes/nodes/mesh/mesh_object_output.py

```diff
@@ -151,6 +151,18 @@ def setMesh(self, outMesh, mesh):
         for attribute in mesh.iterVertexColorAttributes():
             vertexColorLayer = outMesh.vertex_colors.new(name = attribute.name, do_init = False)
             vertexColorLayer.data.foreach_set("color", attribute.data.asMemoryView())
+        
+        # Vertex Groups
+        vertexAmount = len(outMesh.vertices)
+        for attribute in mesh.iterVertexWeightAttributes():
+            try:
+                vertexGroup = object.vertex_groups[attribute.name]
+            except:
+                vertexGroup = object.vertex_groups.new(name = attribute.name)
+            weights = attribute.data
+            for i in range(vertexAmount):
+                vertexGroup.add([i], weights[i], "REPLACE") 
```

Comment:
Would `vertexGroup.add(list(range(vertexAmound)), weights, "REPLACE")` be faster?

Reply:
That is not possible, Vertex Weight does not have `foreach_set/foreach_get` functionality.

Reply:
But `add` can add multiple elements together, can't it? So the above statement should work?

Reply:
It can set the single weight to multiple vertices, however, it does not set weights to individual vertices.
If I add that line, shows this error: `TypeError: VertexGroup.add(): error with argument 2, "weight" -  Function.weight expected a float type`.

Reply:
I see. Ignore this comment then.